api/graphql/schema.graphql

```diff
@@ -41,6 +41,8 @@ type User {
   id: ID!
   sub: String
   roles: [Role]
+  createdDate: DateTime @rename(attribute: "created_at")
+  updatedDate: DateTime @rename(attribute: "updated_at")
```

Comment:
May I ask why you didn't go with `createdAt`/`updatedAt`?

Reply:
Good question. I used a naming convention from a few examples that I found for `createdDate` in [this PR](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5116/files#diff-f673486caba3bebff8c413229d032f383f75536fef46f4196a35ecc5760994daR304) from last week that I applied to the current PR: 
https://github.com/GCTC-NTGC/gc-digital-talent/blob/39cf708cff33319e357f7acc7d5ea007a9f33d27/api/graphql/schema.graphql#L564
https://github.com/GCTC-NTGC/gc-digital-talent/blob/39cf708cff33319e357f7acc7d5ea007a9f33d27/api/graphql/schema.graphql#L569

Digging a bit deeper now, it seems there are other examples of database columns `*_at` (although none of the `created_at` or `updated_at` Laravel [timestamp](https://laravel.com/docs/9.x/eloquent#timestamps) columns) that do not follow this convention:
https://github.com/GCTC-NTGC/gc-digital-talent/blob/39cf708cff33319e357f7acc7d5ea007a9f33d27/api/graphql/schema.graphql#L333
https://github.com/GCTC-NTGC/gc-digital-talent/blob/39cf708cff33319e357f7acc7d5ea007a9f33d27/api/graphql/schema.graphql#L414-L415